locations/templates/quarkus-application/template.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           title: Name
           type: string
           description: Unique name of the application
-          default: my-quarkus-app
+          default: my-quarkus-app-job
```

Comment:
why?

Reply:
This is not a `job`, so the suffix here is missleading.

Reply:
When the job will work as github workflow, I will clean the template of this PR. Please wait

Reply:
Name reverted to `default: my-quarkus-app`